src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+const PRICE_FOR_DAY = 40;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 50;
```

Comment:
The value for `SHORT_TERM_DISCOUNT` is incorrect. According to the task description, the discount for renting the car for 3 or more days should be $20, not $50.